CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - taint-mode: Sanitizers that match exactly a source or a sink are filtered out,
   making it possible to use `- pattern: $F(...)` for declaring that any other
   function is a sanitizer
+- Semgrep now scans executable scripts whose shebang interpreter matches the
+  rule's language
```

Comment:
There's no need to inspect the files for a shebang line because semgrep-core already does it. It's in `Guess_lang`: https://github.com/returntocorp/semgrep/blob/develop/semgrep-core/src/core/Guess_lang.ml

Reply:
Yes, but without this logic, the file names never get passed to semgrep-core.

Reply:
Seems really unfortunate that we do this logic twice

Reply:
yep :(